CHANGELOG.md

```diff
@@ -1,10 +1,9 @@
-# Change Log
+# Backpack Android changelog
 
-All notable changes to this project will be documented in this file.
+[Unreleased changes](./UNRELEASED.md).
 
-## Master
-
-_Nothing Yet_
+## 0.2.1
+Test Release
```

Comment:
What is dis?

Reply:
I think the description is missing and that should be a placeholder, I'll update it shortly 😅